### DISCUSSION TITLE
Does a dedicated build server/instance make sense?

### DISCUSSION THREAD

**Discussion by hmontazeri** [coollabsio/coolify#983] [Q&A]
I'm wondering. While it's nice to save costs and run everything on 1 machine, it would be pretty cool to be able to use a dedicated build server, so the CPU won't spike on the app/service runner server. 

The performance of the apps on the server during build time is pretty bad 😢 

What do you all think?

**Comment by peaklabs-dev** [comment]
@hmontazeri It makes sense if you are building a lot of heavy applications, but in general a coolify server and a server that hosts all your resources (or one powerful server) is enough for most users.
